pkg/vcs/linux.go

```diff
@@ -304,7 +305,7 @@ func ParseMaintainersLinux(text []byte) Recipients {
 
 const configBisectTag = "# Minimized by syzkaller"
 
-func (ctx *linux) Minimize(target *targets.Target, original, baseline []byte,
+func (ctx *linux) Minimize(target *targets.Target, original, baseline []byte, typ dashapi.CrashType,
```

Comment:
Consider adding a description. If I follow this does three things:
1. Disable / Modify certains kconfs based on kernel version tag
2. Disable Instrumentation kconfigs, if instrumentation is not needed for crash
3. Optionally minimize via binary search against smaller baseline config